mgmt/rest/common/common.go

```diff
@@ -0,0 +1,39 @@
+package common
+
+import (
+	"io/ioutil"
+	"os"
+	"path"
+	"runtime"
+
+	log "github.com/Sirupsen/logrus"
+	"github.com/intelsdi-x/snap/control"
+)
+
+func WriteFile(filename string, b []byte) (string, error) {
+	// Create temporary directory
+	dir, err := ioutil.TempDir(control.GetDefaultConfig().TempDirPath, "snap-plugin-")
```

Comment:
Ok so the path for the plugin that's written will be something like:
`<TempDirPath>/snap-plugin-<unique identifier>/snap-plugin-<plugin-name>`? And the reason for separate directories per plugin is that multiple plugins might have the same binary name?